pkgs/tools/admin/lxd/default.nix

```diff
@@ -42,6 +14,47 @@ buildGoModule rec {
     sha256 = "sha256-mYyDYO8k4MVoNa8xfp1vH2nyuhNsDJ93s9F5hjaMftk=";
   };
 
+  binPath = [
+    acl
+    attr
+    bash
+    btrfs-progs
+    criu
+    dnsmasq
+    gnutar
+    gzip
+    iproute2
+    iptables
+    rsync
+    squashfsTools
+    xz
```

Comment:
I suggest adding some more packages to this list

- `util-linux` I needed this to even create a VM instance, otherwise it errors trying to for `prlimit`
- `gptfdisk` VM extraction tries to use `sfdisk`
- `nftables` for firewall support when nftables is enabled